### PR TITLE
Ep3: Change cp -r exercise to create directory structure

### LINKED ISSUE #839: Episode 3: Incorrect wording of learning objectives
Unix Shell: Working with Files and Directories

Within the overview section, it may be useful to either clarify the use of "diagram" in the objective statement  "Create a directory hierarchy that matches a given diagram." this is unclear for learners and is not used throughout the lesson.

### LINKED ISSUE #897: Episode 3: Copy folder exercise - poor example to be setting?
#890 gave rise to this idea.
Re: `copy a folder structure but not the files`
http://swcarpentry.github.io/shell-novice/03-create/index.html#copy-a-folder-structure-but-not-the-files

I think it's a clunky horrible solution to the problem, and not a great workflow to be recommending -- imagine if there are lots of files to be copied only to then be deleted. How about rewording the question to challenge the learner to recreate the folder structure using `mkdir` commands?

This better matches the learning objective
> Create a directory hierarchy that matches a given diagram.

### PR DESCRIPTION
**Pull request by gcapes**
Fix #897.
Fix #839.

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -776,51 +776,62 @@ Oftentimes one needs to copy or move several files at once. This can be done by
...
-> > followed by the recursive copy command which would copy them.
+> > The first two sets of commands achieve this objective.
+> > The first set uses relative paths to create the top level directory before
+> > the subdirectories.
 > >
-> > The third set of commands would achieve the objective, but in a time-consuming way:
-> > the first command copies the directory recursively, but the second command deletes
-> > interactively, prompting for confirmation for each file and directory.
+> > The third set of commands will give an error because `mkdir` won't create a subdirectory
+> > of a non-existant direcotory: the intermediate level folders must be created first.
```
Comment (gdevenyi) [inline _episodes/03-create.md]: direcotory -> directory

**Comment by gdevenyi** [comment]
Much improved compared to the copy and delete.

**Comment by gcapes** [comment]
All good now?
